lib/licenses.nix

```diff
@@ -149,6 +149,12 @@ in mkLicense lset) ({
     fullName = " BitTorrent Open Source License v1.1";
   };
 
+  binaryninja = {
+    fullName = "Binaryninja Non-Commercial License Agreement";
+    url = "https://docs.binary.ninja/about/license.html";
+    free = false;
+  };
```

Comment:
We can probably just inline this in the package, since it's unlikely to ever be used in any other package.

Reply:
fixed